spec/rails_helper.rb

```diff
@@ -19,4 +19,5 @@
   config.filter_rails_from_backtrace!
   config.include FactoryBot::Syntax::Methods
   config.include(Shoulda::Matchers::ActiveRecord, type: :model)
+  config.include ResponseJSON
```

Comment:
You need to create this module in your support folder

Reply:
created